src/components/PostsList.tsx

```diff
@@ -1,54 +1,54 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-
 import classNames from 'classnames';
 import React from 'react';
-import { Post } from '../types/Post';
-
-type Props = {
-  posts: Post[];
-  selectedPostId?: number;
-  onPostSelected: (post: Post | null) => void;
-};
+import { useAppDispatch, useAppSelector } from '../app/hooks';
+import * as selectedPostActions from '../features/selectedPost';
```

Comment:
Ensure that the import path '../features/selectedPost' correctly points to the file where the `selectedPostActions` are defined. If the path is incorrect, it will cause runtime errors.